intro/intro.rst

```diff
@@ -229,20 +308,39 @@ Getting help by using the **?** operator after an object:
 	sep:  string inserted between values, default a space.
 	end:  string appended after the last value, default a newline.
 
+.. seealso:: 
+
+    * IPython user manual: http://ipython.org/ipython-doc/dev/index.html
+
+    * Jupyter Notebook QuickStart:
+      http://jupyter.readthedocs.io/en/latest/content-quickstart.html
 
-Elaboration of the algorithm in an editor
+Elaboration of the work in an editor
 ..........................................
 
-Create a file `my_file.py` in a text editor. Under EPD (Enthought Python
-Distribution), you can use `Scite`, available from the start menu. Under
-Python(x,y), you can use Spyder. Under Ubuntu, if you don't already have your
-favorite editor, we would advise installing `Stani's Python editor`. In the
-file, add the following lines::
+As you move forward, it will be important to not only work interactively,
+but also to create and reuse Python files. For this, a powerful code editor
+will get you far. Here are several good easy-to-use options:
```

Comment:
Beyond general-purpose text editors, here are several good easy-to-use options
(not so sure about this modification, just a suggestion)